doc/tutorial.rst

```diff
@@ -586,8 +573,9 @@ The `static/js/src/simplevideo_edit.js` JavaScript file should look like this: :
           maxwidth: $(element).find('input[name=maxwidth]').val(),
           maxheight: $(element).find('input[name=maxheight]').val()
         };
+        runtime.notify('save', {state: 'start'});
         $.post(handlerUrl, JSON.stringify(data)).done(function(response) {
-          window.location.reload(false);
+          runtime.notify('save', {state: 'end'});
```

Comment:
@Kelketek Not sure why it's there either. I guess it tries to introduce the concept of children, but since it doesn't really explain it in detail I find it rather confusing.